src/EnergyPlus/SurfaceGeometry.cc

```diff
@@ -14032,66 +14035,53 @@ namespace SurfaceGeometry {
+            // thisMaterial->Porosity = 0.0;
             thisMaterial->Resistance = 0.0;
             thisMaterial->SpecHeat = 0.0;
-            thisMaterial->ThermGradCoef = 0.0;
+            // thisMaterial->ThermGradCoef = 0.0;
             thisMaterial->Thickness = distance;
-            thisMaterial->VaporDiffus = 0.0;
-            thisMaterial->gasTypes = Material::GasType::Custom;
-            thisMaterial->GasCon = 0.0;
-            thisMaterial->GasVis = 0.0;
-            thisMaterial->GasCp = 0.0;
-            thisMaterial->GasWght = 0.0;
-            thisMaterial->GasFract = 0.0;
-            thisMaterial->gasTypes(1) = Material::GasType::Air;
-            thisMaterial->GlassSpectralDataPtr = 0;
-            thisMaterial->NumberOfGasesInMixture = 1;
-            thisMaterial->GasCon(1, 1) = 2.873e-3;
-            thisMaterial->GasCon(2, 1) = 7.760e-5;
-            thisMaterial->GasVis(1, 1) = 3.723e-6;
-            thisMaterial->GasVis(2, 1) = 4.940e-8;
-            thisMaterial->GasCp(1, 1) = 1002.737;
-            thisMaterial->GasCp(2, 1) = 1.2324e-2;
-            thisMaterial->GasWght(1) = 28.97;
-            thisMaterial->GasFract(1) = 1.0;
+            // thisMaterial->VaporDiffus = 0.0;
+            // thisMaterial->GlassSpectralDataPtr = 0;
+            thisMaterial->numGases = 1;
+            thisMaterial->gases[0] = Material::gases[(int)Material::GasType::Air];
+            thisMaterial->gasFracts[0] = 1.0;
             thisMaterial->AbsorpSolar = 0.0;
             thisMaterial->AbsorpThermal = 0.0;
             thisMaterial->AbsorpVisible = 0.0;
             thisMaterial->Trans = 0.0;
             thisMaterial->TransVis = 0.0;
-            thisMaterial->GlassTransDirtFactor = 0.0;
-            thisMaterial->ReflectShade = 0.0;
-            thisMaterial->ReflectShadeVis = 0.0;
-            thisMaterial->AbsorpThermalBack = 0.0;
-            thisMaterial->AbsorpThermalFront = 0.0;
-            thisMaterial->ReflectSolBeamBack = 0.0;
-            thisMaterial->ReflectSolBeamFront = 0.0;
-            thisMaterial->ReflectSolDiffBack = 0.0;
-            thisMaterial->ReflectSolDiffFront = 0.0;
-            thisMaterial->ReflectVisBeamBack = 0.0;
-            thisMaterial->ReflectVisBeamFront = 0.0;
-            thisMaterial->ReflectVisDiffBack = 0.0;
-            thisMaterial->ReflectVisDiffFront = 0.0;
-            thisMaterial->TransSolBeam = 0.0;
-            thisMaterial->TransThermal = 0.0;
-            thisMaterial->TransVisBeam = 0.0;
-            thisMaterial->BlindDataPtr = 0;
-            thisMaterial->WinShadeToGlassDist = 0.0;
-            thisMaterial->WinShadeTopOpeningMult = 0.0;
-            thisMaterial->WinShadeBottomOpeningMult = 0.0;
-            thisMaterial->WinShadeLeftOpeningMult = 0.0;
-            thisMaterial->WinShadeRightOpeningMult = 0.0;
-            thisMaterial->WinShadeAirFlowPermeability = 0.0;
+            // thisMaterial->GlassTransDirtFactor = 0.0;
```

Comment:
None of these are relevant to `Material::GasMixture`. Can delete. Should delete.